From e28620a11646b1962a713a070a448e2d79fc1729 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Wed, 17 Mar 2021 19:01:56 +0200
Subject: [PATCH] [SW-30985] gaudi2_arc_gnu: change syscfg default value

For Onix, target, align the default value to the same
as that of ASIC

Change-Id: I991e3efd7cbc3354f14e638c02b802cd60e35b35
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arc_gnu/onix.c b/gaudi2_arc_gnu/onix.c
index ee75b58..e08debd 100644
--- a/gaudi2_arc_gnu/onix.c
+++ b/gaudi2_arc_gnu/onix.c
@@ -21,11 +21,7 @@
 #define UART_CLOCK_FREQUENCY	SYS_CLOCK_FREQUENCY
 #define PLDM_UART_BAUDRATE	57600
 
-#define EFUSE_SYSCFG_ONIX_DEFAULT \
-				(EFUSE_SYSCFG_FW_VERSION_LOCATION_SPI | \
-				 EFUSE_SYSCFG_KEY_LOCATION_SPI | \
-				 EFUSE_SYSCFG_REVOKATION_LOCATION_SPI | \
-				 EFUSE_SYSCFG_ENABLE_VALIDATION_MASK)
+#define EFUSE_SYSCFG_ONIX_DEFAULT	0
 
 u32 hl_skip_pcie_init(void)
 {
